docs/concepts/why-filecoin.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Why Filecoin?
+---
+
+# Why Filecoin?
+
+## Open market
+
+In Filecoin, file storage and retrieval is negotiated in open markets. Anybody can join the Filecoin network without needing permission. Running a miner only requires an internet connection and spare disk space. By lowering the barriers to entry, Filecoin enables a thriving ecosystem of many independent storage providers.
+
+## Competitive prices
+
+Prices for storage and retrieval are determined by supply and demand, not corporate pricing departments. Filecoin makes reliable storage available at hypercompetitive prices. Miners compete based on their storage, reliability and speed rather than through marketing or attempting to lock users in.
+
+## Reliable storage
+
+Because storage is paid for, Filecoin provides a viable economic reason for files to stay available over time. Files are stored on computers that are reliable and well-connected to the internet. Although Filecoin is a peer-to-peer network, it does not rely solely on an ephemeral swarm of peers for files to stay online.
```

Comment:
"an ephemeral swarm of peers" is a phrase folks can't understand without a fair amount of pre-existing knowledge of P2P. Do you mean that you don't have to have a ton of people online simultaneously?

Reply:
Yes I agree this sentence was confusing and wasn’t adding much useful information so I removed it.